pytket/pytket/_tket/circuit.pyi

```diff
@@ -2705,7 +2705,7 @@ class CustomGateDef:
         """
 class DiagonalBox(Op):
     """
-    A box for synthesising a diagonal unitary matrix into a sequence of multiplexed-Rz gates.
+    A box for synthesising a diagonal unitary matrix into a sequence of multiplexed-Rz gates. Implementation based on Theorem 7 of https://arxiv.org/abs/quant-ph/0406176. The decomposed circuit has at most 2^n-2 CX gates.
```

Comment:
A box for synthesising an n-qubit diagonal unitary matrix into a sequence of multiplexed-Rz gates. Implementation based on Theorem 7 of arxiv.org/abs/quant-ph/0406176. The decomposed circuit has at most 2^n-2 CX gates.